python-lib/font_exceptions_dict.py

```diff
@@ -1,27 +1,36 @@
 # -*- coding: utf-8 -*-
-"""Module with constants defining the wordcloud font exceptions for languages not supported by Noto Sans Display"""
+"""Module with constants defining the wordcloud font exceptions for languages not supported by the default font.
+- Default font is NotoSansMerged-Regular-1000upem.ttf, which is a merged font based on various NotoSans fonts:
+    - NotoSansDisplay-Regular
+    - NotoSansArabic-Regular
+    - NotoSansArmenian-Regular
+    - NotoSansBengali-Regular
+    - NotoSansDevanagari-Regular
+    - NotoSansHebrew-Regular
+    - NotoSansSinhala-Regular
+    - NotoSansTamil-Regular
+    - NotoSansThai-Regular
+
+- NotoSansMerged-Regular-2048upem.ttf is a similar font containing fonts that couldn't be merged with precited ones:
+    - NotoSansGujarati-Regular
+    - NotoSansKannada-Regular
+    - NotoSansMalayalam-Regular
+    - NotoSansTelugu-Regular
+
+- NotoSansCJKsc-Regular.otf couldn't be merged with other fonts as it contains too many different characters
```

Comment:
Does this font also work with English? I expect we will sometimes have the case of mixed English / CJK languages in the same dataset

Reply:
yes, it's already the case in test data

Reply:
Super